_opensearch/supported-field-types/xy-point.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: xy point
+nav_order: 58
+has_children: false
+parent: Cartesian field types
+grand_parent: Supported field types
+---
+
+# xy point field type
+
+An xy point field type contains a point in a two-dimensional Cartesian coordinate system, specified by x and y coordinates. It is based on the Lucene [XYPoint](https://lucene.apache.org/core/9_3_0/core/org/apache/lucene/geo/XYPoint.html) field type. The xy point field type is similar to the [geopoint]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point/) field type, but does not have the range limitations of geopoint. The coordinates of an xy point are single-precision floating point values. For information about the range and precision of floating-point values, see [Numeric field types]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/numeric/).
```

Comment:
Can you please check if this link is working. 
[geopoint]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/geo-point/)

I'm getting a 404 error when I'm trying to access it

Reply:
This is a relative link. It is working, but you cannot access it directly. We have a link checker and I made sure that all links are working as part of the commit.

Reply:
Got it, Thanks!